networking.adoc

```diff
@@ -0,0 +1,1081 @@
+      "<ip_address>"
+      ]
+  }'
+----
+
+
+[source,terminal]
+----
+$ oc patch hostsubnet <node_name> --type=merge -p \ 
+  '{
+    "egressIPs": [ 
+      "<ip_address_1>",
+      "<ip_address_N>"
+      ]
+  }'
+----
+
+
+[source,terminal]
+----
+$ oc patch hostsubnet node1 --type=merge -p \
+  '{"egressIPs": ["192.168.1.100", "192.168.1.101", "192.168.1.102"]}'
+----
+
+## Configuring an egress firewall for a project
+
+[source,terminal]
+----
+$ oc create -f <policy_name>.yaml -n <project>
+----
+
+## Configuring an egress router pod destination list from a config map
+
+[source,terminal]
+----
+$ oc delete configmap egress-routes --ignore-not-found
+----
+
+
+[source,terminal]
+----
+$ oc create configmap egress-routes \
+  --from-file=destination=my-egress-destination.txt
+----
+
+## Configuring kube-proxy
+
+[source,terminal]
+----
+$ oc edit network.operator.openshift.io cluster
+----
+
+## Deploying an egress router pod in DNS proxy mode
+
+[source,terminal]
+----
+$ oc create -f egress-router-service.yaml
+----
+
+## Disabling multicast for a project
+
+[source,terminal]
+----
+$ oc annotate netnamespace <namespace> \ 
+    netnamespace.network.openshift.io/multicast-enabled-
+----
+
+## Editing an egress firewall for a project
+
+[source,terminal]
+----
+$ oc replace -f <filename>.yaml
+----
+
+## Enabling multicast for a project
+
+[source,terminal]
+----
+$ oc annotate netnamespace <namespace> \
+    netnamespace.network.openshift.io/multicast-enabled=true
+----
+
+## Configuring network isolation using OpenShift SDN
+
+[source,terminal]
+----
+$ oc adm pod-network join-projects --to=<project1> <project2> <project3>
+----
+
+
+[source,terminal]
+----
+$ oc adm pod-network isolate-projects <project1> <project2>
+----
+
+
+[source,terminal]
+----
+$ oc adm pod-network make-projects-global <project1> <project2>
+----
+
+## Removing an egress firewall from a project
+
+[source,terminal]
+----
+$ oc delete -n <project> egressnetworkpolicy <name>
+----
+
+## Assigning an egress IP address
+
+[source,terminal]
+----
+$ oc apply -f <egressips_name>.yaml 
+----
+
+## Configuring an egress firewall for a project
+
+[source,terminal]
+----
+$ oc create -f <policy_name>.yaml -n <project>
+----
+
+## Configuring an egress IP address
+
+[source,terminal]
+----
+$ oc label nodes <node_name> k8s.ovn.org/egress-assignable="" 
+----
+
+## Disabling multicast for a project
+
+[source,terminal]
+----
+$ oc annotate namespace <namespace> \ 
+    k8s.ovn.org/multicast-enabled-
+----
+
+## Editing an egress firewall for a project
+
+[source,terminal]
+----
+$ oc replace -f <filename>.yaml
+----
+
+## Enabling multicast for a project
+
+[source,terminal]
+----
+$ oc annotate namespace <namespace> \
+    k8s.ovn.org/multicast-enabled=true
+----
+
+## Migrate from the OpenShift SDN cluster network provider
+
+[source,terminal]
+----
+$ oc annotate Network.operator.openshift.io cluster \
+  'networkoperator.openshift.io/network-migration'=""
```

Comment:
Requires a declarative alternative

Reply:
@siamaksade, from this comment down through the next 10 or so, these all relate to a single procedure for migrating from SDN to OVN-K.

I began following the pattern that is established--add a TIP to follow the imperative--and the experience is terrible.  I don't plan to act on your comments in this area (nw-ovn-kubernetes-migration.adoc) because it's a one-off procedure that is not routine and the customer experience would be choppy and terrible.